signer/signer_trust.go

```diff
@@ -30,10 +31,14 @@ type NotarySigner struct {
 func NewNotarySigner(hostname string, port string, tlscafile string) *NotarySigner {
 	var opts []grpc.DialOption
 	netAddr := net.JoinHostPort(hostname, port)
-	creds, err := credentials.NewClientTLSFromFile(tlscafile, hostname)
+	tlsConfig, err := utils.ConfigureClientTLS(&utils.ClientTLSOpts{
+		RootCAFile: tlscafile,
```

Comment:
Even if it didn't, it would probably make sense to be able to load multiple CAs from a single file for the root CA pool, as you suggested for the client CA pool.  Thanks for pointing out this backwards-incompatible change!